### ISSUE TITLE
Bridge breaking in forbidden jungle

### ISSUE THREAD

**Issue by JammishW** [open-goal/jak-project#1705]
### Acknowledgements

- [X] I have [searched](https://github.com/open-goal/jak-project/issues?q=is%3Aissue+is%3Aopen+label%3Abug+sort%3Aupdated-desc) for duplicate or closed bug reports
- [X] I understand that I am supposed to provide my own legitimately obtained copy of the game

### Describe the Bug

The bridge over the fisherman starts wobbling out of control if you look at it, worsening over time.

![fig 1](https://user-images.githubusercontent.com/110031705/180996142-9e342f50-30d8-448c-825c-e338632185f2.png)
![fig 2](https://user-images.githubusercontent.com/110031705/180996233-2fd09862-97e0-454e-b19a-2e3bfaf3220c.png)
 
This is not just a visual bug, it does affect collision. When it breaks completly the planks are fully vertical, rendering the bridge impassable until it is fixed by loading in again.

From my uneducated guess, purely formed by basic experimentation, it seems to be caused by an issue with the bridge clipping into the ground weirdly, as shown below.

![clipping](https://user-images.githubusercontent.com/110031705/180997603-cd9287f7-ac0a-4337-b43f-04385e50e2ac.png)

### How To Reproduce

Cross the bridge over the fisherman. Make sure to walk over the last few planks. You may need to hit them a few times. turn the camera around to look at it. It seemed to happen most when looking from the angle of the second image, but as long as it's in view it seems to be able to do this.

### Does this problem occur on original hardware or PCSX2?

Not needed, bug is obvious

### Expected Behavior

When I step off the bride it should stop moving.

### Operating System

Windows 10

### OpenGOAL Version

0.1.21

### Game Version

NTSC 1.0 (black label)

### Are you using a high refresh-rate monitor (>60Hz)

No

**Comment by Zedb0T** [comment]
This is actually a bug in the original game! Also it is not unique to Jak and daxter (probably) or this bridge in general. When the game runs At unintended frame rates, the physics that make the bridge bounce and snap back to position mess up and cause the bridge to exponentially freak out. This can also be observed in sandover, snowy mountain, and red dead redemption 2.

**Comment by Zedb0T** [comment]
No not a problem at all! It's pretty neat and extremely rare on ps2 (since the game was kind of made to run on ps2 hardware but it does happen).

It's pretty cool and fun to play around with. If you run into anything else strange please continue to open issues. Helps the devs fix as much as possible!

If you want help looking at your settings to see if we can make it less common for you @zed in the opengoal discord

**Comment by JammishW** [comment]
will do, joined the server

**Comment by ManDude** [comment]
I want to look into this regardless as ot's pretty annoying.

**Comment by water111** [comment]
the thing that makes it go crazy is uneven frametimes, like if the game is alternating between 60 and 30 fps

**Comment by mikaelyusifov** [comment]
I tried going into the settings, turning on the FPS limiter and setting it to 30 fps, but it didn't fix the issue. The bridge physics mess up so bad, it starts engulfing the entire area with stretched polygons.

There were other issues such as the game speed constantly fluctuating, but the 30 fps lock seems to have fixed that. It doesn't look great, but it's the only way I found to make the  game playable. I assume there is a general FPS issue that I don't know how to fix, as I haven't seen many other comments about unplayable fps fluctuation.
